packages/dds/merge-tree/src/test/testClient.ts

```diff
@@ -361,21 +357,15 @@ export class TestClient extends Client {
         });
 
         assert(segment !== undefined, "No segment found");
-
         const segoff = this.getSlideToSegment({ segment, offset }) ?? segment;
+        if (segoff.segment === undefined || segoff.offset === undefined) {
+            return DetachedReferencePosition;
+        }
 
-        const slowPathResult =
-            segoff.segment !== undefined
-            && segoff.offset !== undefined
-            && this.findReconnectionPosition(segoff.segment, localSeq) + segoff.offset;
-
-        assert.equal(fastPathSegment, segoff.segment ?? undefined, "Unequal rebasePosition computed segments");
-        assert.equal(fastPathResult, slowPathResult, "Unequal rebasePosition results");
-        return fastPathResult;
+        return this.findReconnectionPosition(segoff.segment, localSeq) + segoff.offset;
```

Comment:
We should probably still validate the slow path vs. fast path here. Rather than validate two numbers are equal though we'd validate that the `segoff` in local scope here matches the `segoff` returned by `findReconnectionPosition`.

Reply:
if you want to do this in a follow-up PR on the same item I'm ok with that as this only affects a small number of tests.